app/components/settings/categories/DisplaySettings.scss

```diff
@@ -0,0 +1,86 @@
+$break-row: 1080px;
```

Comment:
I'm not a big fan of all the hard-coded stuff in this file. Why do we need these instead of setting up the flex container so all of this is handled for us?

Reply:
My idea is here is to always display 3 theme items per row and move the next items to the next row. And at 1080px and smaller since the layout starts to not look good it will then respond to that size and become mobile / tablet layout and display 1 theme per row with a vertical scrollbar.

I will humbly accept any criticism / suggestions 
Thanks
Clark